packages/components/src/Drawer/__snapshots__/Drawer.test.js.snap

```diff
@@ -341,53 +336,46 @@ exports[`Drawer should render with tabs specific actions by tab with selectedTab
             label="ActionCenter-tab-2"
           >
             <withI18nextTranslation(ActionButton)
+              available={true}
+              bsStyle="default"
+              disabled={false}
               id="view-center-tab-2"
+              inProgress={false}
               label="ActionCenter-tab-2"
+              loading={false}
+              t={[Function]}
+              tooltipPlacement="top"
             >
-              <ActionButton
-                available={true}
+              <Button
+                active={false}
+                aria-label="ActionCenter-tab-2"
+                block={false}
+                bsClass="btn"
                 bsStyle="default"
+                className=""
                 disabled={false}
-                i18n={Object {}}
                 id="view-center-tab-2"
-                inProgress={false}
-                label="ActionCenter-tab-2"
-                loading={false}
-                t={[Function]}
-                tReady={true}
-                tooltipPlacement="top"
+                onClick={[Function]}
+                onMouseDown={[Function]}
+                role={null}
```

Comment:
This change is here because of the talend/script i18n HOC mock. It returns the original component instead of wrapping it.

Before 
```
<withI18nextTranslation(ActionButton)>
    <ActionButton>
        ... // children
    </ActionButton>
</withI18nextTranslation(ActionButton)>
```

After
```
<withI18nextTranslation(ActionButton)>
    ... // children
</withI18nextTranslation(ActionButton)>
```